### PR TITLE
Use struct instead of bytes conversion

### LINKED ISSUE #1462: Use struct.pack and struct.unpack instead of int.to_bytes and int.from_bytes
In bugbug/test_scheduling.py.

The changes can simply be tested by running the tests/test_test_scheduling.py tests.

### PR DESCRIPTION
**Pull request by yabirgb**
Fixes #1462 

What I did was to use [struct](https://docs.python.org/3.7/library/struct.html) as mentioned in the issue to make the conversion

### CODE REVIEW

**Comment by marco-c** [comment]
Thanks, the patch looks good overall, but linting is failing. Check the failure on Taskcluster by clicking the `Details` link.
You can also run linting locally by using pre-commit.

**Comment by yabirgb** [comment]
@marco-c Done. Sorry but yesterday the service was under maintenance and I couldn't see what was happening

**Comment by marco-c** [comment]
There is yet another usage to update in the file, in the `get_touched_together` function.

File: bugbug/test_scheduling.py [hunk 1]
```diff
@@ -125,11 +126,11 @@ def set_touched_together(f1, f2):
     key = get_touched_together_key(f1, f2)
 
     if key not in touched_together:
-        touched_together[key] = (1).to_bytes(4, sys.byteorder)
+        touched_together[key] = struct.pack("i", 1)
```
Comment (marco-c) [inline bugbug/test_scheduling.py:129]: It's always positive, so you can use an unsigned int (`I`).

Comment (yabirgb) [reply bugbug/test_scheduling.py:129]: In the `else` is also always positive?

Comment (marco-c) [reply bugbug/test_scheduling.py:129]: Yep, it's the same number, in the else it's just increased by 1 :)

**Comment by marco-c** [comment]
> There is yet another usage to update in the file, in the get_touched_together function.

Don't forget ^

**Pull request merged**